src/witan/app/handler.clj

```diff
@@ -44,7 +44,10 @@
 (defn add-token!
   [user-id tokens]
   (let [token (user/random-token)
-        ttl (t/days 28)]
+        ttl (t/days 28)
+        existing (some #(if (= user-id (-> % second :user)) %) @tokens)]
+    (when existing ;; remove an existing token for this user
+      (swap! tokens dissoc (first existing)))
```

Comment:
So this is just the authn tokens. @elisehuard and I spoke about this and reasoned that for the MVP we could "get away with" leaving them here, but as soon as we do any kind of load balancing this will die, so we need an idea of how to scale this solution at some point.